keys/generate

```diff
@@ -4,13 +4,18 @@ set -e -u
 
 cd $(dirname $0)
 
-docker run --rm -v $PWD/web:/keys concourse/concourse \
+WORK_DIR="$PWD"
+if [ "$MSYSTEM" == "MINGW64" ]; then
```

Comment:
Getting the following error on Ubuntu and macOS:
```
./keys/generate: line 8: MSYSTEM: unbound variable
```

Reply:
This is due to the `-u` flag. Resolving by loading the MSYSTEM env variable before setting `-u`, to avoid false positives on the `$PWD` var, but we could also just move the `set -u` statement to after the `if`.